src/snowflake/connector/connection.py

```diff
@@ -620,6 +631,9 @@ def __preprocess_auth_instance(self, auth_instance):
         if type(auth_instance) is AuthByWebBrowser:
```

Comment:
Can you rename this to AuthByExtBrowser please. This has a potential to cause confusion.

Reply:
Created a ticket https://snowflakecomputing.atlassian.net/browse/SNOW-232776 to do it later!